test/testother.cpp

```diff
@@ -1551,14 +1551,15 @@ class TestOther : public TestFixture {
         ASSERT_EQUALS("", errout.str());
 
         check("class X {\n"
-              "    uint64_t i;\n"
+              "    void* i;\n"
               "};\n"
               "class Y : X {\n"
-              "    uint64_t j;\n"
+              "    void* j;\n"
+              "    void* k;\n"
               "};\n"
               "void f(X x, Y y) {\n"
               "}");
-        ASSERT_EQUALS("[test.cpp:7]: (performance) Function parameter 'y' should be passed by reference.\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:8]: (performance) Function parameter 'y' should be passed by reference.\n", errout.str());
```

Comment:
yes it seems to me your size guesses are better than the old. if you add a new test instead of rewriting the old one I think this should go in.